styles/tuta/catppuccin.user.css

```
@@ -2,7 +2,7 @@
 @name Tuta Catppuccin
 @namespace github.com/catppuccin/userstyles/styles/tuta
 @homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/tuta
-@version 0.0.9
+@version 0.1.0
```

Technically, since this is a patch of assorted fixes, according to [Semver](https://semver.org/), we would increase the third number ("PATCH version when you make backward compatible bug fixes"). Semver doesn't mean much in userstyles but it is good to learn :)
```suggestion
@version 0.0.10
```